pixie/vm/stdlib.py

```diff
@@ -105,8 +105,14 @@ def default_str(x):
     assert isinstance(tp, Type)
     return rt.wrap(u"<inst " + tp._name + u">")
 
+def default_hash(x):
+    tp = x.type()
```

Comment:
Do we need this? It's going to get removed by the JIT (or even RPython) but it is extra code.